driver/csiplugin/controllerserver.go

```diff
@@ -2169,12 +2168,24 @@ func (cs *ScaleControllerServer) DeleteVolume(newctx context.Context, req *csi.D
 			}
 		}
 
+		// Check if fileset exists and the creator is IBM Storage Scale CSI driver
+		filesetInfo, err := conn.ListFileset(ctx, FilesystemName, FilesetName)
+		loggerId := utils.GetLoggerId(ctx)
+		if err != nil {
+			klog.Errorf("[%s]  unable to list fileset [%v] in filesystem [%v]. Error: %v", loggerId, FilesetName, FilesystemName, err)
+			return nil, status.Error(codes.Internal, fmt.Sprintf("unable to list fileset [%v] in filesystem [%v]. Error: %v", FilesetName, FilesystemName, err))
+		} else if !reflect.ValueOf(filesetInfo).IsZero() && filesetInfo.Config.Comment != connectors.FilesetComment {
+			klog.V(4).Infof("Fileset [%v] is not created by IBM Container Storage Interface driver, skipping the fileset delete", FilesetName)
```

Comment:
can we have this as a default level log(klog.Infof) instead of klog.V(4) so that it get logged in all cases

Reply:
@deeghuge could you please help review this PR ?